OWML.ModHelper.Menus/ModComboInput.cs

```diff
@@ -39,10 +39,9 @@ public ModComboInput(TwoButtonToggleElement element, IModMenu menu, IModInputMen
 
         private void UpdateLayout(string currentCombination)
         {
-            var childCount = _layoutGroup.transform.childCount;
-            for (var i = childCount - 1; i >= 0; i--)
+            foreach (Transform child in _layoutGroup.transform)
             {
-                GameObject.Destroy(_layoutGroup.transform.GetChild(i).gameObject);
+                GameObject.Destroy(child.gameObject);
```

Comment:
please don't. That's just you making things annoying either to me or to yourself. this part is moved to LayoutManager and tidied up there

Reply:
I don't agree that it's unnecessary but I get that it's annoying since it conflicts with your PR, reverting it 👍